plugins/action_change_status/src/changeStatus.ts

```diff
@@ -131,8 +133,9 @@ const changeReviewStatus = async (
       returnObject.error_log = 'Review Status not changed'
       returnObject.output = {
         status: newStatus,
-        statusId: currentStatus.status_history_id,
+        statusId: currentStatus.id,
         reviewStatusHistoryTimestamp: currentStatus.time_created,
+        reviewId,
```

Comment:
Small tidy-ups in "changeStatus" action to make sure returned output was consistent whether or not the status actually changed.